docs/import/manage_disaster.js

```diff
@@ -404,22 +416,29 @@ function initializeScoreSelectors(states) {
   // Initialize headers.
   removeAllButFirstFromRow(headerRow);
   for (const state of states) {
-    headerRow.append(createTd().html(state + ' Assets'));
+    headerRow.append(createTd().text(state + ' Assets'));
   }
 
   // For each asset type, add select for all assets for each state.
   for (const scoreAssetType of scoreAssetTypes) {
-    const id = assetSelectionRowPrefix + scoreAssetType[0];
+    const type = scoreAssetType[0];
+    const id = assetSelectionRowPrefix + type;
     const propertyPath = scoreAssetType[1];
+    const expectedColumns = scoreAssetType[3];
     const row = $('#' + id);
     removeAllButFirstFromRow(row);
     for (const state of states) {
       if (stateAssets.get(state)) {
         const statePropertyPath = propertyPath.concat([state]);
-        row.append(createTd().append(addAssetDataChangeHandler(
+        const select =
```

Comment:
nit: can leave this as a variable for readability, but could also inline now

Reply:
I think helpful for readability so going to leave.